machina/templates/machina/forum_member/forum_profile_update.html

```diff
@@ -32,6 +32,18 @@ <h3 class="panel-title">{% trans "Edit forum profile" %}</h3>
                         </div>
                         <div class="row">
                             <div class="col-md-12">
+                                {% include "partials/form_field.html" with field=form.auto_subscribe_topics %}
+                            </div>
+                        </div>
+                        {% if MACHINA_ENABLE_EMAIL_NOTIFICATIONS %}
```

Comment:
True. I'll override the forms `__init__`  and conditionally remove the setting if email is not enabled.